src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findTab } from '../../utils';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const currentTab = findTab(tabs, selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
```

Comment:
If you don't need a `tab` object, you can do destructuring directly in the callback arguments.

Reply:
i need tab in onClick event --  onClick={() => selectedTab(tab, id)}